agent/php_rinit.c

```diff
@@ -104,9 +108,18 @@ PHP_RINIT_FUNCTION(newrelic) {
     char* str = (char*)e;
     nr_free(str);
   }
+  void zval_stack_dtor(void* e, NRUNUSED void* d) {
+    zval* zv = (zval*)e;
+    nr_php_zval_free(&zv);
+  }
   nr_stack_init(&NRPRG(predis_ctxs), NR_STACK_DEFAULT_CAPACITY);
+  nr_stack_init(&NRPRG(wordpress_tags), NR_STACK_DEFAULT_CAPACITY);
```

Comment:
Does nr_stack_destroy_fields deinit this?
Or does it also need to be deinited in rshutdown?

Reply:
The stack itself is a vector wrapper with the actual vector being a member `elements` that is a pointer to the stack. `nr_stack_init` allocates this memory and sets `elements` accordingly. `nr_stack_destroy_fields` then iterates over all elements in the `elements` vector, calling the dtor on each, before finally freeing the actual vector. No additional cleanup should be needed beyond this.